lua_modules/ds3231/ds3231.lua

```diff
@@ -5,6 +5,8 @@
 -- Tobie Booth <tbooth@hindbra.in>
 --------------------------------------------------------------------------------
 
+require("bit")
```

Comment:
Got it, thanks. Would you then only require the non-obvious ones i.e. that's the reason you don't require `i2c`?